docs/mvvm/source-gen/diagnostics/MVVMTK0015.md

```diff
@@ -0,0 +1,68 @@
+---
+title: MVVM Toolkit error MVVMTK0015
+author: Sergio0694
+description: MVVM Toolkit error MVVMTK0015
+keywords: community toolkit, dotnet, csharp, mvvm, net core, net standard, source generators
+dev_langs:
+  - csharp
+---
+
+# MVVM Toolkit error MVVMTK0015
+
+The target(s) of `[NotifyPropertyChangedFor]` must be a (different) accessible property in its parent type.
+
+The following sample generates MVVMTK0015:
+
+```csharp
+using CommunityToolkit.Mvvm.ComponentModel;
+
+namespace MyApp;
+
+public partial class SampleViewModel : ObservableObject
+{
+    // The target property name is invalid (cannot be null)
+    [ObservableProperty]
+    [NotifyPropertyChangedFor(null)]
```

Comment:
Is this a useful example? Maybe put the invalid property name one first?

Reply:
Yeah agreed. Swapped the two in 54c89a38a79adf2e0ba6b7e48276e6dec49643ea 🙂